abs/snowball-c-git/PKGBUILD

```diff
@@ -23,7 +23,6 @@ build() {
 
 package() {
     cd $srcdir/$pkgname
-    install -Dm644 include/* $pkgdir/usr/include/libstemmer.h
```

Comment:
We can remove this package and the `groonga` package can depend on the `libstemmer` package instead of this package.